lib/rmg/RmgArea.h

```diff
@@ -30,9 +30,9 @@ namespace rmg
 	public:
 		Area() = default;
 		Area(const Area &);
-		Area(const Area &&);
-		Area(const Tileset & tiles);
-		Area(const Tileset & relative, const int3 & position); //create from relative positions
+		Area(const Area &&) noexcept;
```

Comment:
Is explicit copy constructor necessary here? 

Similarly, is explicit move constructor necessary here? 
Also, shouldn't it be defined as `Area(Area &&)`?

Reply:
It is move constructor. It should not be const at theory, but it was const, clang-tidy just added noexcept here.

Reply:
Yeah, not questioning your change, but rather *really* weird code both before & after.

Reply:
removed const, unsure what more should I do manually.